scalableminds/webknossos
# Create unified json format for ObjectId

When writing an objectId to json (as to string), the result is now just a string (or rather JsString), no longer an `{"id":"myId"}` object.
ObjectId can now also be read from json strings directly..
When reading from Json, it is also validated against the mongo id schema.

I believe we could now replace most occurences of `id.id` by just `id` (not only in json, but also in both sql and strings), but I did not want to create too many line changes in this pr, since there are many other open PRs. Let’s just replace those gradually when editing different areas of the code.

### URL of deployed dev instance (used for testing):
- https://objectidjson.webknossos.xyz

### Steps to test:
- Create a project, save, should work.
- in dataset settings, change allowed teams, should work

### Issues:
- fixes #6375 

---
